nix/dapp.nix

```diff
@@ -0,0 +1,740 @@
+# This file has been generated by dapp2nix 0.0.2, do not edit.
+{ solidityPackage, solc, dapp2 }:
+
+let
+  inherit (builtins) map listToAttrs attrNames attrValues;
+  mapAttrs = if (builtins ? mapAttrs)
```

Comment:
Maybe add a comment here in dapp2nix that says "This is to ensure backwards-compatibility with nix < 2.1"?

Reply:
Actually this is [defined in nixpkgs](
https://github.com/asymmetric/nixpkgs/tree/2ec5e9595becf05b052ce4c61a05d87ce95d19af/lib/attrsets.nix#L207) already, so you won't need it.

Reply:
Didn't want to depend on `lib` but could change it.

Reply:
True. Up to you.